tools/ci_build/github/azure-pipelines/templates/jar-maven-signing-linux.yml

```diff
@@ -0,0 +1,64 @@
+parameters:
+  - name: JarFileDirectory
+    type: string
+
+  - name: WorkingDirectory
+    type: string
+
+  - name: EnableJarMavenSigning
```

Comment:
It looks  EnableJarMavenSiging are always true.
Can we set the default value as true?
Or 
Why we need this parameter?
Does it useless

Reply:
1. set its default value to true.
2. reason: in case the signing process hit any issues, may break the following pipeline running, can change to manually signing in this case.